### DISCUSSION TITLE
First map after launching osu!lazer has stutter/freezes.

### DISCUSSION THREAD

**Discussion by KoleckOLP** [ppy/osu#25501] [Q&A] [answered]
Hello, my osu lazer installation is on an SSD but my maps are on an HDD.
and every time I open lazer the first map has significant freezes.
So, what I do it enable autoplay and scroll to the end of the map and then back to the start,
then I remove autoplay and I don't have freezes for the rest of the osu session.
I didn't open an issue about it because it might be a known issue but if it's not can this discussion be turned into an issue?

**Comment by peppy** [comment, marked as answer]
Being tracked via https://github.com/ppy/osu/issues/26535
